wdl/ExpansionHunterDenovo.wdl

```diff
@@ -0,0 +1,347 @@
+## ExpansionHunter denovo (EHdn)
+##
+## This WDL implements workflows for
+## EHdn's two execution modes:
+## (i) case-control analysis;
+## (ii) outliner analysis.
+
+version 1.0
+
+import "Structs.wdl"
+
+struct FilenamePostfixes {
+  String locus
+  String motif
+  String profile
+  String merged_profile
+  Int profile_len
+}
+
+workflow EHdnSTRAnalysis {
+
+  input {
+    String analysis_type
+    String str_comparison_type
+    Array[File] case_reads_filenames
+    Array[File] case_indexes_filenames
```

Comment:
I would make the index column optional and infer index URIs if not provided - see `Module00a`'s `bam_or_cram_index` input as an example.

Reply:
Thanks for the suggestion, so even if a file is defined in the body of a workflow, Cromwell will localize that file.